### PR TITLE
Create Microsoft Defender AV details.yaml

### PR DESCRIPTION
**Pull request by sonalimeshram21**
A list of all devices with below AV information

   Required items, please complete
   
   Change(s):
   - Adding a new query to fetch Microsoft defender AV details

   Reason for Change(s):
   - Introduced a new feature for showing up to date reports in Device health tab. Customers are asking for AH queries as well.

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - Yes ran the query on required DB and table using the same KQL query

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Hunting Queries/Microsoft 365 Defender/TVM/Microsoft Defender AV details.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  // Author: Sonali Meshram
+  // @someshram
+  // ------------------------
+  // 1.	A list of all devices with below AV information
+  // 2.	This query will identify the Microsoft Defender Antivirus Security Intelligence version, Security Intelligence up to date value, Engine version, Engine up to date value,  Product version (aka Platform version),Product (aka Platform) up to date value, Security Intelligence publish/build timestamp, Security intel refresh timestamp
+requiredDataConnectors:
+- connectorId: MicrosoftThreatProtection
+  dataTypes:
+  - DeviceTvmInfoGathering
+query: |
```
Comment (aprakash13) [inline Hunting Queries/Microsoft 365 Defender/TVM/Microsoft Defender AV details.yaml:13]: Please add Tactics and relevant technique and if it is not applicable, just leave it as blank. i.e 

tactics:[]
relevantTechniques:[]

**Review by aprakash13 — CHANGES_REQUESTED** [review]
Thanks for the contributions. 
Provided a few small template related suggestions. Would you please make the same changes with the other submissions in the PR as well.

File: Hunting Queries/Microsoft 365 Defender/TVM/Microsoft Defender AV details.yaml (hunk 1 above)
Comment (sonalimeshram21) [reply Hunting Queries/Microsoft 365 Defender/TVM/Microsoft Defender AV details.yaml:8]: resolved

**Comment by sonalimeshram21** [comment]
> Thanks for the contributions. Provided a few small template related suggestions. Would you please make the same changes with the other submissions in the PR as well.

Resolved the comments

**Review by aprakash13 — APPROVED** [review]

**Pull request merged**